packages/odyssey-react/src/components/Icon/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+export { default as Anchor } from './Anchor'
```

Comment:
+1  the rexported syntax here - I was even discussing the idea (with @arnoldsandoval-okta ) of removing all default exports from our system so that code is more discoverable and all identifiers need to be explicitly aliased 

Incidentally this does not need to be a `.tsx` file and can simply be `.ts`

Reply:
While technically true, the file is autogenerated and the configuration only allows for the output of a single file type https://react-svgr.com/docs/options/